osm2streets-java/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -ex
+
+javac -d build src/*.java -target 11 -source 11
+cargo build
```

Comment:
Of course you'll want `--release` here and below for anything on Maven, or if you're not making changes on this end frequently